subsys/bluetooth/mesh/Kconfig

```diff
@@ -344,16 +344,13 @@ config BT_MESH_MSG_CACHE_SIZE
 	  for each received network PDU and increases RAM footprint proportionately.
 
 config BT_MESH_ADV_BUF_COUNT
-	int "Number of advertising buffers"
+	int "Number of advertising buffers for local messages"
 	default 6
 	range 1 256
 	help
-	  Number of advertising buffers available. This should be chosen
-	  based on what kind of features the local node should have. E.g.
-	  a relay will perform better the more buffers it has. Another
-	  thing to consider is outgoing segmented messages. There must
-	  be at least three more advertising buffers than the maximum
-	  supported outgoing segment count (BT_MESH_TX_SEG_MAX).
+	  Number of advertising buffers available for sending local messages.
+	  This should be chosen based on the number of local messages that the node
+	  can send at concurrently.
```

Comment:
```suggestion
	  can send concurrently.
```

Reply:
Maybe we should use simultaneously after all..? I tried to glance through the file, to get context to this config option, and I see that simultaneous(ly) is widely used :)

Reply:
PR to address this comment: https://github.com/zephyrproject-rtos/zephyr/pull/50407. @mia-ko please have a look.